packages/firestore/src/local/simple_db.ts

```diff
@@ -406,6 +406,18 @@ export interface IterateOptions {
   reverse?: boolean;
 }
 
+/** An error that wraps exceptions that thrown during IndexedDB execution. */
+export class IndexedDbTransactionError extends FirestoreError {
+  name = 'IndexedDbTransactionError';
+
+  constructor(cause: Error) {
+    super(
+      Code.UNKNOWN,
```

Comment:
`UNAVAILABLE` could be a better code for this given what we know about when this happens.